lib/routes/deepseek/news.ts

```diff
@@ -0,0 +1,96 @@
+import { Route, Data, DataItem } from '@/types';
+import ofetch from '@/utils/ofetch';
+import { load } from 'cheerio';
+import cache from '@/utils/cache';
+
+const ROOT_URL = 'https://api-docs.deepseek.com/zh-cn';
+
+const NEWS_LIST_SELECTOR = 'ul.menu__list > li:nth-child(2) ul > li.theme-doc-sidebar-item-link';
+const ARTICLE_CONTENT_SELECTOR = '.theme-doc-markdown > div > div';
+const ARTICLE_TITLE_SELECTOR = ARTICLE_CONTENT_SELECTOR + ' > h1';
+
+
+// 获取消息列表 / get article list
+const fetchPageContent = async (url: string) => {
+    const response = await ofetch(url);
+    return load(response);
+};
+
+// 提取正文内容 / extract article content
+const extractArticleInfo = ($article: cheerio.Root, pageURL: string) => {
+    const contentElement = $article(ARTICLE_CONTENT_SELECTOR);
+    const title = $article(ARTICLE_TITLE_SELECTOR).text();
+    $article(ARTICLE_TITLE_SELECTOR).remove();  // 移除标题，避免重复 / remove title to avoid duplication
+    const content = contentElement.html();
+    return { title, content, pageURL };
+};
+
+const parseDateString = (dateString: string) => {
+    const pubDate = new Date(dateString);
+    return pubDate.toUTCString();
+};
+
+
+// 创建消息 / create article
+const createDataItem = (item: cheerio.Element, $: cheerio.Root): Promise<DataItem> => {
+    const $item = $(item);
+    const link = $item.find('a').attr('href');
+    const dateString = $item.find('a').text().split(' ').at(-1);
+    const pageURL = new URL(link || '', ROOT_URL).href;
+
+    return cache.tryGet(
+        pageURL,
+        async () => {
+            const $article = await fetchPageContent(pageURL);
+            const { title, content } = extractArticleInfo($article, pageURL);
+            const pubDate = parseDateString(dateString);
+
+            return {
+                title,
+                link: pageURL,
+                pubDate,
+                description: content || undefined,
+            };
+        },
+        21600 // 6 小时 / 6 hours
```

Comment:
```suggestion
```
No need to hard-coded cache duration as it should be managed by the default value (config.cache.contentExpire)